docs/en/getting-started/example-datasets/opensky.md

```diff
@@ -19,7 +19,7 @@ https://doi.org/10.5194/essd-13-357-2021
 Run the command:
 
 ```bash
-wget -O- https://zenodo.org/record/5092942 | grep -oP 'https://zenodo.org/record/5092942/files/flightlist_\d+_\d+\.csv\.gz' | xargs wget
+wget -O- https://zenodo.org/record/5092942 | grep -oP 'https://zenodo.org/records/5092942/files/flightlist_\d+_\d+\.csv\.gz' | xargs wget
```

Comment:
Which MacOS do you use?
I think I can try to reproduce and fix it

Reply:
@yariks5s. Thanks for your reply. I updated these URLs and please help me to check that.

I appreciate your help.

Reply:
``` shell
yaroslav@Yaroslavs-MacBook-Pro logs % sudo wget -O- https://zenodo.org/records/5092942 | grep -oP 'https://zenodo.org/records/5092942/files/flightlist_\d+_\d+\.csv\.gz' | xargs wget
grep: invalid option -- P
usage: grep [-abcdDEFGHhIiJLlMmnOopqRSsUVvwXxZz] [-A num] [-B num] [-C[num]]
	[-e pattern] [-f file] [--binary-files=value] [--color=when]
	[--context[=num]] [--directories=action] [--label] [--line-buffered]
	[--null] [pattern] [file ...]
Password:
--2024-04-09 13:26:15--  https://zenodo.org/records/5092942
Resolving zenodo.org (zenodo.org)... 188.184.98.238, 188.185.79.172, 188.184.103.159
Connecting to zenodo.org (zenodo.org)|188.184.98.238|:443... connected.
HTTP request sent, awaiting response... 200 OK
Length: 969927 (947K) [text/html]
Saving to: ‘STDOUT’

-                                                                     0%[                                                                                                                                                                  ]       0  --.-KB/s    in 0s


Cannot write to ‘-’ (Success).
```

macOS Sonoma 14.4

Reply:
@yariks5s, thanks for your reply.

The problem is about `-P` option. This option is available for `GNU grep` and it's `BSD grep` in the MacOS.

Finally, I figure out solutions and it can run in the Linux-like and Unix-like operating systems.

Please run the following commands to verify that:

```sh
$ sw_vers
ProductName:    macOS
ProductVersion: 11.6
BuildVersion:   20G165
$ wget -O- https://zenodo.org/records/5092942 | grep -oE 'https://zenodo.org/records/5092942/files/flightlist_[0-9]+_[0-9]+\.csv\.gz' | xargs wget

```

Reply:
now this script works good, thanks🙂